.gitignore

```diff
@@ -1,2 +1,3 @@
-out
+out/**/*.js
+out/**/*.map
```

Comment:
actually it does use the .yaml files, they are the linting rules.  i had to change gitignore to persist them. happy if you have another way to do this.